tests/server_notices/test_resource_limits_server_notices.py

```diff
@@ -321,7 +327,9 @@ def test_invite_with_notice(self):
 
         # Scan the events in the room to search for a message from the server notices
         # user.
-        events = channel.json_body["rooms"]["join"][room_id]["timeline"]["events"]
+        events = channel.json_body["rooms"][Membership.JOIN][room_id]["timeline"][
```

Comment:
as a matter of general preference, I would avoid using symbolic constants like this in tests where possible. It's a chance to spot typos in the definition of the constant!

Reply:
That's a good point! I also did so when adding the tests for knocking, though that made a bit more sense when we needed to switch the prefix from unstable to stable.

But a good tactic to carry forward.